src/views/Characters/CharacterDetails.test.jsx

```diff
@@ -0,0 +1,18 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+
+import CharacterDetails from './CharacterDetails';
+-describe('Character Details', () => {
+  it('Should render one character by id', async () => {
+    render(
+      <MemoryRouter>
+        <CharacterDetails />
+      </MemoryRouter>
```

Comment:
Instead of rendering CharacterDetails here, we will want to render App and have an initialEntries prop on our memory router component.